polar-language-server/Makefile

```diff
@@ -3,8 +3,8 @@
 CARGO_FLAGS ?= --dev
 
 build:
-	wasm-pack --quiet build $(CARGO_FLAGS) --target nodejs --out-dir ../vscode/oso/out
-	rm -f ../vscode/oso/out/.gitignore ../vscode/oso/out/package.json
+	wasm-pack --quiet build $(CARGO_FLAGS) --target nodejs --out-dir $(OUT_DIR)
+	rm -f $(OUT_DIR)/.gitignore $(OUT_DIR)/package.json
```

Comment:
should we add a default like `CARGO_FLAGS` above?

Reply:
Or, alternatively, blow up loudly & predictably if `OUT_DIR` isn't set?

Reply:
I had a default at first (see commit before "fix" commit) but removed it because moving forward I think it won't make sense to have one -- I'll make it blow up loudly and predictably